<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WishesWorkersWidget</class>
 <widget class="QWidget" name="WishesWorkersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>WishesWorkersWidget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Worker</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="p_lbl_Surname">
        <property name="text">
         <string>Surname</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="p_lbl_FirstName">
        <property name="text">
         <string>FirstName</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="p_line_ID_worker"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="p_line_Surname">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="p_lbl_ID_worker">
        <property name="text">
         <string>ID worker</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="p_lbl_LastName">
        <property name="text">
         <string>LastName</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="p_line_FirstName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="p_line_LastName">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Single wish</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="2" column="0">
       <widget class="QLabel" name="p_lbl_EndDate">
        <property name="text">
         <string>EndDate</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="p_lbl_MinWorkerDays">
        <property name="text">
         <string>Minimum number of working days</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QSpinBox" name="p_spinBox_MinWorkerDays">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="p_lbl_ID_wishes">
        <property name="text">
         <string>ID wishes</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="p_lbl_StartDate">
        <property name="text">
         <string>Start date</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QTimeEdit" name="p_timeEdit_GetStartedLaterThan">
        <property name="wrapping">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDateTimeEdit" name="p_dateTimeEdit_EndDate">
        <property name="wrapping">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="p_lbl_Position">
        <property name="text">
         <string>Position</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="p_lbl_MaxWorkerDays">
        <property name="text">
         <string>Maximum number of working days</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="p_spinBox_MaxWorkerDays">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QComboBox" name="p_comboBox_Position"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="p_lbl_Addition">
        <property name="text">
         <string>Addition</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="p_line_ID_wishes"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="p_lbl_GetStartedLaterThan">
        <property name="text">
         <string>Start work not earlier ...</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="p_lbl_GetEndEarlierThan">
        <property name="text">
         <string>End of work no later than ...</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QTimeEdit" name="p_timeEdit_GetEndEarlierThan">
        <property name="wrapping">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDateTimeEdit" name="p_dateTimeEdit_StartDate">
        <property name="wrapping">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPlainTextEdit" name="p_TextEdit_Addition"/>
      </item>
      <item row="9" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>List of wishes</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="p_list_Wishes"/>
      </item>
      <item>
       <widget class="QPushButton" name="p_button_AddWishes">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="p_button_DeleteWishes">
        <property name="text">
         <string>Delete</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="p_button_CreateNew">
       <property name="text">
        <string>Create New...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="p_button_Open">
       <property name="text">
        <string>Open...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="p_button_Save">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="p_button_SaveAs">
       <property name="text">
        <string>Save as...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>p_line_ID_worker</tabstop>
  <tabstop>p_line_Surname</tabstop>
  <tabstop>p_line_FirstName</tabstop>
  <tabstop>p_line_LastName</tabstop>
  <tabstop>p_line_ID_wishes</tabstop>
  <tabstop>p_dateTimeEdit_StartDate</tabstop>
  <tabstop>p_dateTimeEdit_EndDate</tabstop>
  <tabstop>p_comboBox_Position</tabstop>
  <tabstop>p_TextEdit_Addition</tabstop>
  <tabstop>p_spinBox_MinWorkerDays</tabstop>
  <tabstop>p_spinBox_MaxWorkerDays</tabstop>
  <tabstop>p_timeEdit_GetStartedLaterThan</tabstop>
  <tabstop>p_timeEdit_GetEndEarlierThan</tabstop>
  <tabstop>p_list_Wishes</tabstop>
  <tabstop>p_button_AddWishes</tabstop>
  <tabstop>p_button_DeleteWishes</tabstop>
  <tabstop>p_button_CreateNew</tabstop>
  <tabstop>p_button_Open</tabstop>
  <tabstop>p_button_Save</tabstop>
  <tabstop>p_button_SaveAs</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
